pkg/authz/requestlogger.go

```diff
@@ -0,0 +1,30 @@
+package authz
+
+import (
+	"net/http"
+	"strings"
+)
+
+func requestLogger(r *http.Request) string {
+	var sb strings.Builder
+
+	sb.WriteString(r.Method)
+	sb.WriteString(" ")
+	sb.WriteString(r.URL.String())
+
+	var bodyBytes []byte
+	if r.Body != nil {
+		defer func() {
+			_ = r.Body.Close()
+		}()
+		bodyBytes = make([]byte, r.ContentLength)
+		_, err := r.Body.Read(bodyBytes)
+		if err != nil {
+			sb.WriteString("Body: <unknown>")
+		} else {
+			sb.WriteString(" Body: " + string(bodyBytes))
```

Comment:
I know that this PR isn't changing this behavior, but the first thing I think when I see request body logging is whether it could leak anything sensitive. Are we worried about that in this case?